frontend/compositions/visualizations/popUps/popUpComp.tsx

```diff
@@ -0,0 +1,39 @@
+import { useCallback, useEffect, useRef, useState } from "react"
+import { getTitleCaseFromCamel } from "../../../helpers"
+import { Coord } from "../utilities/chartTypes"
+import styles from "./popUps.module.css"
+
+export interface PopUpProps {
+  shouldShowPopUp?: boolean
+  headerText?: string
+  bodyText?: string
+  location?: Coord
+}
+
+export default function PopUp({ shouldShowPopUp, headerText, bodyText, location }: PopUpProps) {
+  const classNames = `${styles.popUp} ${shouldShowPopUp ? styles.popUpOpen : styles.popUpClosed}`
+
+  const ariaLabel: string = `Detailed information about ${headerText}`
+  const popUpBodyId: string = `${headerText}Details`
+
+  const style = {
+    transform: `translate(${location?.x}px, ${location?.y}px) translateX(50%)`
+  }
+
+  return (
+    <div
+      id={"popup"}
+      className={classNames}
+      style={style}
+      aria-label={ariaLabel}
+      aria-describedby={popUpBodyId}
+      role="tooltip">
+      <header id="head" className={styles.popUpHead}>
+        {headerText}
+      </header>
```

Comment:
We won't want to use `header` here - that is a page landmark element that needs to be a direct sibling element to our `main` tag (like the header of the whole website/page)

Instead, this should probably be a "heading" element (h1-6) - which need to follow chronological order, not skipping a number, top-down on the page